Tasks/PowerShellOnTargetMachines/make.json

```diff
@@ -4,5 +4,34 @@
             "module": "../Common/Deployment/TelemetryHelper",
             "type": "ps"
         }
-    ]
+    ],
+    "externals": {
+        "archivePackages": [
+            {
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/DeploymentUtilities/M118.1/DeploymentUtilities.zip",
+                "dest": "./"
+            }
+        ],
+        "nugetv2": [
+            {
+                "name": "VstsTaskSdk",
```

Comment:
why do you need this?

Reply:
PS3 releated utility methods are in this module e.g Find-vstsfiles,Get-VstsLocString etc  so it's required for PS3.